tests/topotests/ospf6_ecmp_inter_area/r7/ospf6d.conf

```diff
@@ -13,9 +13,6 @@ interface r7-eth2
  ipv6 ospf6 hello-interval 2
  ipv6 ospf6 dead-interval 10
 !
-interface r7-eth3
- shutdown
-!
```

Comment:
Not having the shutdown in ospf6d.conf makes sense, but instead of removing it, it probably should get moved to zebra.conf. Same for r8.